blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -0,0 +1,130 @@
+
+## How we use Gather
+We started using Gather exactly six weeks ago. Your first task is to create your own space. Luckily, they offer plenty of pre-made spaces to pick from. I created a virtual office space, where we could sit at desks or meet at the bar in a kitchen.
+
+![](images/gather/gather-templates.png)
+_Pick an existing gather space from plenty of templates_
+
+From the beginning, I forced myself to keep Gather running every day to act as a role model. Occasionally, I would invite someone from the team to try out Gather with me. 
+
+### All-hands meetings
+Once I reached a "critical mass" of people using Gather, I decided to switch our all-hands meeting on Mondays from Zoom to Gather. We would run our meeting on a small, sunny island. That was also the first time I had to use the [mapmaker](https://support.gather.town/help/mapmaker). Because everything is possible in the Metaverse I decided to build a suburban station to connect the island with our office.
+
+![](images/gather/gather-smm.jpg)
+_Our Satellytes folks gathering for our all-hands meeting on a Monday_
+
+The actual meeting is comparable to Zoom or Meet. We broadcast some news and people can share thoughts and ask questions. The time before and after that meeting is where the Metaverse shines ✨. 
+
+People bump into each other, create random groups, and tell their stories from the weekend. Compare this with the Zoom dystopia, where everyone usually joins into a single room and leave directly after the meeting back into isolation (I exaggerated a little) 
+
+### Pair programming
+A similar effect happens when you sit in the virtual office and suddenly two folks group up on the other side of the table. Instantly you know, _"oh, they do some pair programming"_. When I asked my colleagues about their session, they said it's a nice way to meet at some actual (although virtual) location and then just _"bump into the other person"_ to start the session. 
+
+This feeling is the promise of the Metaverse. It brings back the informal human interactions and spontaneity. You don't have to create a video meeting upfront or send some specific invitation. Who would do that in real life? Instead, you just walk over to the desk of your colleague and start talking.  
+
+![](images/gather/gather-pair.gif)
+
+_Our colleagues Jesús & Felix having a pair programming session_
+
+### Random bumps
+When you walk to someone, while approaching, the video and audio slow fade in. That technique is called "spatial audio technology" and mimics the way we bump into other people in real life, where someone's voice is louder while you are near them, and softer as you walk away. This is wonderful because it feels so familiar and natural.
+
+<video preload autoplay loop width="100%">
+  <source src="images/gather/video.mp4" type="video/mp4" />
+  <source src="images/gather/video.mp4" type="video/mp4" />
+</video>
+
+_Official Gather video from [gather.town](http://www.gather.town)_
+
+### Daily
```

Comment:
"Daily routine"? "Hanging around"? 
At first I though about the scrum artifact, when I read "Daily", tbh

Reply:
good point 🙏